stdpopsim/catalog/HomSap/dfes.py

```diff
@@ -147,3 +147,59 @@ def _HuberLogNormalDFE():
 
 
 _species.add_dfe(_HuberLogNormalDFE())
+
+
+def _KyriazisDFE():
+    id = "Mixed_K23"
+    description = "Deleterious Gamma DFE with additional lethals"
+    long_description = """
+    Return neutral and negative MutationType()s representing a human DFE.
+    Kyriazis et al. (2023), https://doi.org/10.1086/726736.
+    This model is similar to the Kim et al. (2017) DFE based on human
+    genetic data, and with dominance distribution from Henn et al. (2016).
+    One difference is an increased deleterious mutation rate per diploid of
+    0.63, informed by the analysis of 30Mb of human coding sequence.
+    The model is also augmented with an additional proportion of 0.3% of
+    recessive lethals, based on the analysis of Wade et al. (2023).
+    """
+    citations = [
+        stdpopsim.Citation(
+            author="Kyriazis et al.",
+            year=2023,
+            doi="https://doi.org/10.1086/726736",
+            reasons={stdpopsim.CiteReason.DFE},
+        )
+    ]
+    neutral = stdpopsim.MutationType()
+    gamma_mean = -0.0131
+    gamma_shape = 0.186
+    coefs = [0.45, 0.2, 0.05, 0]
+    breaks = [0.001, 0.01, 0.1]
+    gamma = stdpopsim.MutationType(
+        dominance_coeff_list=coefs,
+        dominance_coeff_breaks=breaks,
+        distribution_type="g",  # gamma distribution
+        distribution_args=[gamma_mean, gamma_shape],
+    )
+    h = 0
+    s = -1
+    lethal = stdpopsim.MutationType(
+        distribution_type="f",  # fixed value
+        distribution_args=[s],
+        dominance_coeff=h,
+    )
```

Comment:
totally a nitpick, but I didn't think we needed `h` and `s` on separate lines? feel free to not keep this suggestion. (The comment is because I had to check the fitness adjustment was `1+s` not `exp(s)`.)